voclabs:~/environment/Docker_Microservice_udacity_4 (master) $ kubectl logs $(kubectl get po -o=name | grep flask)
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 100-543-651
[2021-10-27 10:23:08,582] INFO in app: JSON payload: {'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2021-10-27 10:23:08,595] INFO in app: Inference payload DataFrame:    CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2021-10-27 10:23:08,607] INFO in app: Scaling Payload:    CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
/usr/local/lib/python3.7/site-packages/sklearn/preprocessing/data.py:645: DataConversionWarning: Data with input dtype int64, float64 were all converted to float64 by StandardScaler.
  return self.partial_fit(X, y)
/app/app.py:19: DataConversionWarning: Data with input dtype int64, float64 were all converted to float64 by StandardScaler.
  scaled_adhoc_predict = scaler.transform(payload)
[2021-10-27 10:23:08,612] INFO in app: output prediction: 
[20.35373177134412]
127.0.0.1 - - [27/Oct/2021 10:23:08] "POST /predict HTTP/1.1" 200 -